zips/zip-0321.rst

```diff
@@ -93,12 +93,13 @@ The following syntax specification uses ABNF [#RFC5234]_.
   zcashurn        = "zcash:" ( zcashaddress [ "?" zcashparams ] / "?" zcashparams )
   zcashaddress    = 1*( ALPHA / DIGIT )
   zcashparams     = zcashparam [ "&" zcashparams ]
-  zcashparam      = [ addrparam / amountparam / memoparam / messageparam / labelparam / reqparam / otherparam ]
+  zcashparam      = [ addrparam / amountparam / assetidparam / memoparam / messageparam / labelparam / reqparam / otherparam ]
   NONZERO         = %x31-39
   DIGIT           = %x30-39
   paramindex      = "." NONZERO 0*3DIGIT
   addrparam       = "address" [ paramindex ] "=" zcashaddress
   amountparam     = "amount"  [ paramindex ] "=" 1*DIGIT [ "." 1*8DIGIT ]
+  assetidparam    = "asset_id"[ paramindex ] "=" 88*88base64url
```

Comment:
Given the constraints of the `Forward Compatibility` section, this must be a `req-` parameter.

```suggestion
  assetidparam    = "req-asset_id"[ paramindex ] "=" 88*88base64url
```

Reply:
`_` is not allowed.